### PR TITLE
Remove names of security tools

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Remove name of Security tools to guard against giving malicious actors specific information and prevent outdated documentation when the tools change. Direct user to Axway's security statement.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | d462084 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-277.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by lbadenhop** [comment]
@robertengaxway and @cbordier, please review.

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/agent_security.md [hunk 1]
```diff
@@ -284,18 +284,21 @@ To remove the clear value from the agent configuration file, create a correspond
...
-* [Golint](https://github.com/golang/lint) - scans the code for possible coding errors or inconsistencies
+SAST (Static Application Security Testing):
 
-* [Fortify](https://www.microfocus.com/en-us/cyberres/application-security/software-security-center) - static code analyzer that searches for violations of security-specific coding rules and guidelines
+* Scan the code for possible coding errors or inconsistencies
+* Look for code reliability issues, security vulnerabilities, and unit test coverage
+* Scan all library dependencies for security vulnerabilities and compliance issues
 
-* [SonarQube](https://www.sonarqube.org/) - looks for code reliability issues, security vulnerabilities, and unit test coverage
+DSAT (Dynamic Application Security Testing):
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connected_agent_common_reference/agent_security.md:297]: typo, "DAST"

**Review by cbordier — APPROVED** [review]
Approved.

**Comment by dfeldick** [comment]
Seems OK. Guess we'll see what SSG says.

**Review by dfeldick — APPROVED** [review]

**Pull request merged**